am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -27,7 +27,9 @@
         Stream.of(CREATE, READ, UPDATE).collect(toSet());
     public static final Set<Permission> CREATE_PERMISSION = Stream.of(CREATE).collect(toSet());
     public static final Set<Permission> READ_PERMISSION = Stream.of(READ).collect(toSet());
-    public static final JsonNode DATA = JsonNodeFactory.instance.objectNode();
+    public static final JsonNode DATA = JsonNodeFactory.instance.objectNode()
+        .put("name", "John")
```

Comment:
is this needed?

Reply:
I would not say needed. It is to make sample data more real.